_gsocproposals/proposal_GoHEPInterpreter.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Launching Wagon, a WebAssembly interpreter in Go
+layout: plain
+project: Go-HEP
+organization: CERN
```

Comment:
project is fine with Go-HEP I believe. Organization would be your home institute, which we'd need to add.